packages/manager/modules/nutanix/src/dashboard/nodes/list/routing.js

```diff
@@ -6,6 +6,34 @@ export default /* @ngInject */ ($stateProvider) => {
       breadcrumb: /* @ngInject */ () => null,
       goToAddNode: /* @ngInject */ ($state) => () =>
         $state.go('nutanix.dashboard.nodes.all.add-nodes'),
+      powerOffNode: /* @ngInject */ ($state) => (nodeName) =>
+        $state.go('nutanix.dashboard.nodes.all.poweroff-node', {
+          node: nodeName,
+        }),
+      installNode: /* @ngInject */ ($state) => (nodeName) =>
+        $state.go('nutanix.dashboard.nodes.all.install-node', {
+          node: nodeName,
+        }),
+      reinstallNode: /* @ngInject */ ($state) => (nodeName) =>
+        $state.go('nutanix.dashboard.nodes.all.reinstall-node', {
+          node: nodeName,
+        }),
+      uninstallNode: /* @ngInject */ ($state) => (nodeName) =>
+        $state.go('nutanix.dashboard.nodes.all.uninstall-node', {
+          node: nodeName,
+        }),
+      terminateNode: /* @ngInject */ ($state) => (nodeName) =>
+        $state.go('nutanix.dashboard.nodes.all.resiliate-node', {
+          nodeName,
+        }),
```

Comment:
Maybe you can rename nodeName param to node to simplify

```suggestion
      terminateNode: /* @ngInject */ ($state) => (nodeName) =>
        $state.go('nutanix.dashboard.nodes.all.resiliate-node', {
          node: nodeName,
        }),
```
```
terminateNode: /* @ngInject */ ($state) => (node) =>
        $state.go('nutanix.dashboard.nodes.all.resiliate-node', {
          node,
        }),
```